static/src/javascripts/projects/common/modules/commercial/hosted-gallery.js

```diff
@@ -275,7 +277,7 @@ define([
                 if(this.useSwipe){
                     this.translateContent(this.index, 0, 100);
                 }
-
+                omniture.trackLinkImmediate(config.page.analyticsName + ' - image ' + this.index);
```

Comment:
can you use a `data-link-name` attribute instead? that still ends up calling `trackLink`, but normalises the name